web_client/panels/AnnotationSelector.js

```diff
@@ -31,7 +31,8 @@ var AnnotationSelector = Panel.extend({
         'click .h-hide-all-annotations': 'hideAllAnnotations',
         'mouseenter .h-annotation': '_highlightAnnotation',
         'mouseleave .h-annotation': '_unhighlightAnnotation',
-        'change #h-toggle-labels': 'toggleLabels'
+        'change #h-toggle-labels': 'toggleLabels',
+        'change #h-annotation-opacity': '_changeGlobalOpacity'
```

Comment:
Updating the layer opacity is fast enough, let's do this as the slider is dragged (add `'input #h-annotation-opacity': '_changeGlobalOpacity'`).